src/components/menu-item/menu-item.styles.ts

```diff
@@ -91,6 +96,17 @@ export default css`
     visibility: visible;
   }
 
+  /* Add elevation and z-index to submenus */
+  sl-popup::part(popup) {
+    box-shadow: var(--sl-shadow-large);
+    z-index: var(--sl-z-index-dropdown);
+    margin-left: var(--submenu-offset);
+  }
+
+  .menu-item--rtl sl-popup::part(popup) {
+    margin-left: calc(-1 * var(--submenu-offset));
```

Comment:
We could, but it would require an extra property because of the way popover works when things go RTL.

Without logical properties (current):

```css
  sl-popup::part(popup) {
    /* Unrelated properties omitted for brevity */
    margin-left: var(--submenu-offset);
  }

  .menu-item--rtl sl-popup::part(popup) {
    margin-left: calc(-1 * var(--submenu-offset));
  }
```

With logical properties (proposed):

```css
  sl-popup::part(popup) {
    /* Unrelated properties omitted for brevity */
    margin-inline-start: var(--submenu-offset);
  }

  .menu-item--rtl sl-popup::part(popup) {
    margin-inline-start: 0;
    margin-inline-end: calc(-1 * var(--submenu-offset));
  }
```

I figured it was better to assign the value to a single property and calculate its negative value for RTL since it's ultimately controlled by the public `--submenu-offset` custom property.